docs/manual/layout/templates/template-assets.de.md

```diff
@@ -22,17 +24,29 @@ Template zu referenzieren:
 <script src="files/myfolder/custom.js"></script>
 ```
 
-Sollen die Dateien stattdessen im HTML-Header eingebunden werden, so lässt sich das mit folgendem PHP-Code im Template
+Sollen die Dateien stattdessen im HTML-Header eingebunden werden, so lässt sich das mit folgendem Angaben im Template
 anweisen:
 
+
+{{< tabs groupId="templateGroup">}}
+{{% tab name="PHP" %}}
+
+
 ```php
 <?php
 // wird in <head> ausgegeben
+
 $GLOBALS['TL_CSS'][] = 'files/myfolder/custom.css|static';
 $GLOBALS['TL_JAVASCRIPT'][] = 'files/myfolder/custom.js|static';
 ?>
 ```
 
 Diese Umsetzung bietet weitere Optionen: Mit Angabe von `|static` werden die Dateien z.&nbsp;B. zu den bestehenden
-Assets eines Seitenlayouts hinzugefügt bzw. zusammengefasst. Eine detaillierte Beschreibung aller Optionen und
-Ausgabeorte findest du in der Entwickler-Dokumentation unter [Adding CSS & JavaScript Assets](https://docs.contao.org/dev/framework/asset-management/).
+Assets eines Seitenlayouts hinzugefügt bzw. zusammengefasst. 
+
+Eine detaillierte Beschreibung aller Optionen und Ausgabeorte findest du in der Entwickler-Dokumentation 
+unter [Adding CSS & JavaScript Assets](https://docs.contao.org/dev/framework/asset-management/).
+
+
+{{% /tab %}}
+{{< /tabs >}}
```

Comment:
There is only one tab, so I think the tab should be removed for now?

Reply:
I removed the tab.